_posts/2021-01-05-Jamulus-Sound-Devices.md

```diff
@@ -19,7 +19,7 @@ Just [edit this page and wait for an approval](https://github.com/jamulussoftwar
 
 ## Audio devices known to work with Jamulus
 
-Your device must support a sample rate of *48kHz (DVD quality)* - which many devices do - and should allow you to listen to the computer’s audio signal. On Windows it’s recommended to choose a device with a *native ASIO driver* - not ASIO4ALL.
+Your device must support a sample rate of *48 kHz (DVD quality)* - which many devices do - and should allow you to listen to the computer’s audio signal. On Windows it’s recommended to choose a device with a *native ASIO driver* - not ASIO4ALL.
```

Comment:
Are we aiming to be consistent with "48000 Hz" vs "48 kHz"?

Reply:
My preference would be to stick to 48 kHz but I agree it might be best to use one or the other.